docs/docs/contributing-guide/slackcoc.md

```diff
@@ -60,7 +60,7 @@ In addition, violations of this code outside our spaces may affect a person’s
 - If violations occur, organizers will take any action they deem appropriate for the infraction, up to and including expulsion.
 
 :::info
-Portions derived from the [Django Code of Conduct](https://www.djangoproject.com/conduct/), [The Rust Code of Conduct](https://www.rust-lang.org/conduct.html) under a Creative Commons Attribution-ShareAlike license.
+Portions derived from the [Django Code of Conduct](https://www.djangoproject.com/conduct/), [The Rust Code of Conduct](https://www.rust-lang.org/conduct.html) and [The Ada Initiative](https://adainitiative.org/2014/02/18/howto-design-a-code-of-conduct-for-your-community/) under a Creative Commons Attribution-ShareAlike license.
```

Comment:
Kindly change the "Tha Ada Initiative" hyperlink to https://adainitiative.org. The existing URL is broken.

Reply:
@tejakummarikuntla  I have made the requested changes, please check it now.

Reply:
Thank you.